modules/uncertainty/man/flux.uncertainty.Rd

```diff
@@ -0,0 +1,33 @@
+% Generated by roxygen2 (4.1.0): do not edit by hand
+% Please edit documentation in R/flux_uncertainty.R
+\name{flux.uncertainty}
+\alias{flux.uncertainty}
+\title{Calculate parameters for heteroskedastic flux uncertainty}
+\usage{
+flux.uncertainty(measurement, QC = 0, flags = TRUE, bin.num = 10,
+  transform = identity, minBin = 5, ...)
+}
+\arguments{
+\item{measurement}{= flux time-series}
+
+\item{QC}{= quality control flag time series (0 = best)}
+
+\item{flags}{= additional flags on flux filtering of PAIRS (length = 1/2 that of the
+time series, TRUE = use).}
+
+\item{bin.num}{= number of bins (default = 10)}
+
+\item{transform}{= transformation of magnitude (default = identity)}
+
+\item{minBin}{= minimum number of points allowed in a bin}
+}
+\value{
+List of parameters from the fitted uncertainty model
+}
+\description{
+Calculate parameters for heteroskedastic flux uncertainty
+}
+\author{
+???
```

Comment:
Most of these functions are written by either me or Carl, but Ryan documented them without know who wrote them, and then ROxygen propagated that into the man files. I'll go clean that up and include it in my next pull request